external-crates/move/crates/move-package/tests/test_runner.rs

```diff
@@ -17,97 +15,61 @@ use move_package::{
 };
 use move_symbol_pool::Symbol;
 use std::{
-    ffi::OsStr,
     fs,
     path::{Path, PathBuf},
 };
 use tempfile::{tempdir, TempDir};
 
-const EXTENSIONS: &[&str] = &["progress", "resolved", "locked", "notlocked", "compiled"];
+pub fn snapshot_path(pkg: &Path, name: &str, kind: &str) -> PathBuf {
+    pkg.join(format!("{name}@{kind}.snap"))
+}
```

Comment:
It appears as though this function is unused. Not sure if you were planning on using this instead of the `snapshot_path` function in `test_dependency_graph` or not.

Reply:
It was used in an earlier commit, missed the cleanup, thanks!